### ISSUE TITLE
does not honor prefix set in /sur/etc/npmrc, always tries to look in /usr/lib/node_modules

### ISSUE THREAD

**Issue by dkebler** [dylang/npm-check#373]
npm-check does not get the globalconfig set in `/usr/ect/npmrc/`.  It keeps looking in `/usr/lib/node_modules`

``` output of npm config ls
; globalconfig /usr/etc/npmrc
prefix = "/opt/node-global-apps"
```

I noted similar issues going back two years.  Looks like using the `global-modules` and NODE_PATH do not consider that someone would be setting their own custom global path.

**Comment by Artoria2e5** [comment]
This shouldn't happen because `global-modules@1.0.0` depends on `global-prefix@^1.0.1`, which does seem to do [look in the prefix-etc npmrc](https://github.com/jonschlinkert/global-prefix/blob/1.0.1/index.js#L38). Maybe it's time to debug-by-printing in these dependencies.

**Comment by dkebler** [comment]
Sadly this seems a dead project.  Still at 5.9 so this issue has not been addressed.
see npm-check-updates for a project that access the correct global store.

**Comment by Artoria2e5** [comment]
Read their code. I am not sure if spawning npm for `npm config get prefix` is the best way to do it. Maybe it is.

**Comment by dkebler** [comment]
I see two new commits by @LinusU, yea!  Maybe my issue can be addressed?  

BTW ncu doesn't have this issue.  It does honor the prefix in `/usr/etc/npmrc` so this is clearly possible.  https://www.npmjs.com/package/npm-check-updates

**Comment by digeomel** [comment]
I agree with @dkebler . I love the interactive UI of npm-check, but until this issue it's fixed, I'm using npm-check-updates for the same reason. And I have read their code, even contributed to the discussion on what's the best way to get the prefix, which apparently is not as obvious as it should be.
